spec/version_action_spec.rb

```diff
@@ -0,0 +1,528 @@
+require 'spec_helper'
```

Comment:
It'd be great to see some input validation tests on this, as well as _possibly_ mocking out the `ios_version_helper` and `android_version_helper` to ensure we're passing the expected arguments. We want to avoid "testing that the code does what the code does", but those tests can have their place.

End-to-end tests also aren't bad necessarily, but if we only rely on them it'll make our test suite a little bit slower, and encode specific assumptions about the underlying implementation into this "controller" object's tests (which could then break if we were to change `android_version_helper`, which is often a sign of a bad test).